packages/strapi-generate-new/lib/create-starter-project.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const execa = require('execa');
+const fs = require('fs-extra');
+const uuid = require('uuid/v4');
+
+module.exports = async function createStarterProject(scope) {
+  console.log('Creating a starter project.');
+
+  // Name of the repository
+  const repository = scope.starter;
+  const destination = scope.rootPath;
+  const destinationName = scope.name;
+
+  // Clone the starter repository
```

Comment:
You should check if the user has git installed and if the url is a either a git url or an http/https one to dowload the repo differently.

Reply:
I don't understand, do we have to download a git repository differently depending on wether it's `https://github.com/strapi/strapi-starter-react-blog.git` or `git@github.com:strapi/strapi-starter-react-blog.git` ?

Reply:
If no git you should download via the archive and unzip it instead of using git clone.

Reply:
Also git cloning like that isn't the right way to do it. You need to shallow clone the repo / rm the .git folder and init a new git repo with a clean initial commit.